Author: aszlig <aszlig@nix.build>
Date:   Fri Mar 30 19:32:26 2018 +0200

    Remove contact form
    
    This submits user data to https://contact.habitica.com/ when the user
    clicks on the contact link within the "Help" menu item.
    
    We don't want that, so let's remove it and the whole contact buttons
    altogether.
    
    Signed-off-by: aszlig <aszlig@nix.build>
    Filename: remove-contact-form.patch

diff --git a/website/client/components/groups/tavern.vue b/website/client/components/groups/tavern.vue
index eb4d3634d..08bb0a384 100644
--- a/website/client/components/groups/tavern.vue
+++ b/website/client/components/groups/tavern.vue
@@ -101,8 +101,6 @@
 
       sidebar-section(:title="$t('helpfulLinks')")
         ul
-          li
-            a(href='', @click.prevent='modForm()') {{ $t('contactForm') }}
           li
            router-link(to='/static/community-guidelines', v-once) {{ $t('communityGuidelinesLink') }}
           li
@@ -429,7 +427,6 @@
 
 <script>
 import { mapState } from 'client/libs/store';
-import { goToModForm } from 'client/libs/modform';
 
 import { TAVERN_ID } from '../../../common/script/constants';
 import worldBossInfoModal from '../world-boss/worldBossInfoModal';
@@ -511,9 +508,6 @@ export default {
     this.group = await this.$store.dispatch('guilds:getGroup', {groupId: TAVERN_ID});
   },
   methods: {
-    modForm () {
-      goToModForm(this.user);
-    },
     toggleSleep () {
       this.$store.dispatch('user:sleep');
     },
diff --git a/website/client/components/header/menu.vue b/website/client/components/header/menu.vue
index 31df3f29c..f61e52ae6 100644
--- a/website/client/components/header/menu.vue
+++ b/website/client/components/header/menu.vue
@@ -56,7 +56,6 @@ div
             a.topbar-dropdown-item.dropdown-item(href="https://trello.com/c/odmhIqyW/440-read-first-table-of-contents", target='_blank') {{ $t('requestAF') }}
             a.topbar-dropdown-item.dropdown-item(href="http://habitica.wikia.com/wiki/Contributing_to_Habitica", target='_blank') {{ $t('contributing') }}
             a.topbar-dropdown-item.dropdown-item(href="http://habitica.wikia.com/wiki/Habitica_Wiki", target='_blank') {{ $t('wiki') }}
-            a.topbar-dropdown-item.dropdown-item(@click='modForm()') {{ $t('contactForm') }}
       .currency-tray.form-inline
         .item-with-icon(v-if="userHourglasses > 0")
           .top-menu-icon.svg-icon(v-html="icons.hourglasses", v-b-tooltip.hover.bottom="$t('mysticHourglassesTooltip')")
@@ -334,7 +333,6 @@ div
 
 <script>
 import { mapState, mapGetters } from 'client/libs/store';
-import { goToModForm } from 'client/libs/modform';
 
 import gemIcon from 'assets/svg/gem.svg';
 import goldIcon from 'assets/svg/gold.svg';
@@ -389,9 +387,6 @@ export default {
     this.getUserGroupPlans();
   },
   methods: {
-    modForm () {
-      goToModForm(this.user);
-    },
     toggleUserDropdown () {
       this.isUserDropdownOpen = !this.isUserDropdownOpen;
     },
diff --git a/website/client/components/static/header.vue b/website/client/components/static/header.vue
index 68d62bd34..d240ae6ac 100644
--- a/website/client/components/static/header.vue
+++ b/website/client/components/static/header.vue
@@ -16,8 +16,6 @@
           a.nav-link(href='http://blog.habitrpg.com/', target='_blank') {{ $t('tumblr') }}
         router-link.nav-item(tag='li', to='/static/press-kit')
           a.nav-link(v-once) {{ $t('presskit') }}
-        router-link.nav-item(tag='li', to='/static/contact')
-          a.nav-link(v-once) {{ $t('contactUs') }}
       ul.navbar-nav.mr-auto(v-else)
         router-link.nav-item(tag='li', to='/register')
           a.nav-link(v-once) {{ $t('getStarted') }}
diff --git a/website/client/router.js b/website/client/router.js
index db2d38cb6..ea2ed6c92 100644
--- a/website/client/router.js
+++ b/website/client/router.js
@@ -14,7 +14,6 @@ const HomePage = () => import(/* webpackChunkName: "entry" */'./components/stati
 const AppPage = () => import(/* webpackChunkName: "static" */'./components/static/app');
 const ClearBrowserDataPage = () => import(/* webpackChunkName: "static" */'./components/static/clearBrowserData');
 const CommunityGuidelinesPage = () => import(/* webpackChunkName: "static" */'./components/static/communityGuidelines');
-const ContactPage = () => import(/* webpackChunkName: "static" */'./components/static/contact');
 const FAQPage = () => import(/* webpackChunkName: "static" */'./components/static/faq');
 const FeaturesPage = () => import(/* webpackChunkName: "static" */'./components/static/features');
 const GroupPlansPage = () => import(/* webpackChunkName: "static" */'./components/static/groupPlans');
@@ -245,7 +244,6 @@ const router = new VueRouter({
         { name: 'app', path: 'app', component: AppPage, meta: {requiresLogin: false}},
         { name: 'clearBrowserData', path: 'clear-browser-data', component: ClearBrowserDataPage, meta: {requiresLogin: false}},
         { name: 'communityGuidelines', path: 'community-guidelines', component: CommunityGuidelinesPage, meta: {requiresLogin: false}},
-        { name: 'contact', path: 'contact', component: ContactPage, meta: {requiresLogin: false}},
         { name: 'faq', path: 'faq', component: FAQPage, meta: {requiresLogin: false}},
         { name: 'features', path: 'features', component: FeaturesPage, meta: {requiresLogin: false}},
         { name: 'groupPlans', path: 'group-plans', component: GroupPlansPage, meta: {requiresLogin: false}},
